felix.go

```diff
@@ -470,7 +470,7 @@ configRetry:
 	dpConnector.Start()
 
 	if policySyncProcessor != nil {
-		log.WithField("managementSocket", configParams.PolicySyncManagementSocketPath).Info(
+		log.WithField("policy sync directory", configParams.PolicySyncPathPrefix).Info(
```

Comment:
camelCase, and this should be the same field name as the above log statement, for easier grepping of logs